airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/streams.py

```diff
@@ -750,6 +781,9 @@ def _get_metrics(self):
     def _get_reporting_dimensions(self):
         result = super()._get_reporting_dimensions()
         result += self.audience_dimensions
+        # In the case of province breakdown, the date dimension is not supported
+        if self.audience_dimensions == ["province_id"] and "stat_time_day" in result:
+            result.remove("stat_time_day")
```

Comment:
@iberchid this part is causing integration test to fail:
```
data = {'ad_id': 1714125051115569, 'dimensions': {'ad_id': 1714125051115569, 'province_id': '5417618'}, 'metrics': {'ad_name'...o anywhere!', 'adgroup_id': 1714125049901106, 'adgroup_name': 'Ad Group20211020010107', ...}, 'province_id': '5417618'}
key = 'stat_time_day'

>   value = reduce(lambda data, key: data[key], path, record)
E   KeyError: 'stat_time_day'
```

Reply:
@marcosmarxm, as mentioned in the [documentation](https://ads.tiktok.com/marketing_api/docs?id=1751454103714818), `province_id` is not compatible with any time dimension:

![image](https://user-images.githubusercontent.com/93732608/234398288-ce585970-fe0d-42a0-a2f2-ac55581d68ce.png)

Reply:
@iberchid not sure if this stream is going to work using the base class with depends on `stat_time_day` field.

Reply:
I have tested it and it worked, but the tests will fail because you expect the `stat_time_day` dimension.